todays-curiosity.js

```diff
@@ -29,24 +27,21 @@ class todaysCuriosity {
     this.brightPixels = brightenPixels(pixels);
   }
   
-  setDivisions(xDivisions = 20, yDivisions = 20) {
-    this.xDivisions = xDivisions;
-    this.yDivisions = yDivisions;
+  loadDefaultValues() {
+    this.xDivisions = 20;
+    this.yDivisions = 20;
+    this.reductionRatio = 0.6;
+    this.xOffset = 0;
+    this.yOffset = 0;
```

Comment:
The settors methods als played the role of loading defaults, so this method fills that gap in their lieu.